### PR TITLE
Set core version in analytics

### LINKED ISSUE #3183: Report what version of core is being used by the current SDK
The current version of core is part of the new metrics.
One way of achieving the goal is to hard-coding the core version in the weaver while doing a new release (on CI).

### PR DESCRIPTION
**Pull request by papafe**
Fixes #3183

### CODE REVIEW

File: Realm/Realm.Weaver/Analytics/Analytics.cs
Comment (nirinchev) [inline Realm/Realm.Weaver/Analytics/Analytics.cs:64]: This can be a const

Comment (nirinchev) [reply Realm/Realm.Weaver/Analytics/Analytics.cs:64]: This should also solve the build issue.

Comment (papafe) [reply Realm/Realm.Weaver/Analytics/Analytics.cs]: Yep

File: Realm/Realm.Weaver/Analytics/Analytics.cs
Comment (nirinchev) [inline Realm/Realm.Weaver/Analytics/Analytics.cs:64]: We can probably fill in the current Core version since the expectation is that there'll be something here.

**Review by nirinchev — APPROVED** [review]

File: Realm/Realm.Weaver/Analytics/Analytics.cs
Comment (papafe) [reply Realm/Realm.Weaver/Analytics/Analytics.cs:64]: But then we don't fully appreciate the regex that works with an empty string too 😢

**Review by LaPeste — APPROVED** [review]

**Comment by papafe** [comment]
@LaPeste I've just merged from your branch because there was a conflict. Given it's approved, feel free to merge it when you think it's appropriate

**Comment by LaPeste** [comment]
> @LaPeste I've just merged from your branch because there was a conflict. Given it's approved, feel free to merge it when you think it's appropriate

The plan is to have a feature branch that's going to be created as soon as my current PR is approved. Any future change about the epic is going to be merged in the feature branch. So, I suppose I'll just wait for the approval of my own.
@nirinchev did I understand correctly?

**Comment by nirinchev** [comment]
Yeah, let's review/merge your branch into a feature branch before we merge more stuff into it to make it easier to review.

**Pull request merged**
